topics/computational-chemistry/tutorials/analysis-md-simulations/tutorial.md

```diff
@@ -0,0 +1,192 @@
+>
+> {% include snippets/import_via_link.md %}
+>
+{: .hands_on}
+
+
+# Analysis with BIO3D
+
+We'll carry out some basic analysis by calculating RMSD, RMSF and PCA. The tools use the [Bio3D package](http://thegrantlab.org/bio3d/index.php), developed by the [Grant lab](http://thegrantlab.org/).
+
+
+## RMSD
+
+RMSD, or root-mean-square deviation, is a standard measure of structural distance between coordinates. It measures the average distance between a group of atoms (e.g. backbone atoms of a protein). If we calculate RMSD between two sets of atomic coordinates - for example, two time points from the trajectory - the value is a measure of how much the protein conformation has changed. [Wikipedia](https://en.wikipedia.org/wiki/Root-mean-square_deviation_of_atomic_positions) provides more information.
+
+> ### {% icon hands_on %} Hands-on: Calculate RMSD
+>
+> **RMSD Analysis** {% icon tool %} with the following parameters:
+>    - {% icon param-file %} *"dcd trajectory input"*: Trajectory file
+>    - {% icon param-file %} *"pdb input"*: Structure file
+>    - *"Select domains"*: `Calpha` (calculate RMSD only for the C-alpha domain of the protein)
+>
+{: .hands_on}
+
+![Snapshot of RMSD plot]({{ site.baseurl }}{% link topics/computational-chemistry/images/RMSD_plot_33.png %} "RMSD plot for a short CBH1 simulation")
+![Snapshot of RMSD histogram]({{ site.baseurl }}{% link topics/computational-chemistry/images/RMSD_Histogram_Plot_34.png %} "RMSD histogram for a short CBH1 simulation")
+
+> > ### {% icon question %} Question
+> >
+> > What do the features in the RMSD plot tell us?
+> >
+> > > ### {% icon solution %} Solution
+> > > The increase in the RMSD plot with time shows the protein steadily deviates from its original conformation.
+> > >
+> > > The three peaks visible in the histogram suggests the presence of three main conformations which are accessed during the trajectory.
+> > {: .solution}
+> {: .question}
+
+
+## RMSF
+
+The root-mean-square fluctuation (RMSF) measures the average deviation of a particle (e.g. a protein residue) over time from a reference position (typically the time-averaged position of the particle). Thus, RMSF analyzes the portions of structure that are fluctuating from their mean structure the most (or least). 
+
+> ### {% icon hands_on %} Hands-on: Calculate RMSF
+>
+> 1. **RMSF Analysis** {% icon tool %} with the following parameters:
+>    - {% icon param-file %} *"dcd trajectory input"*: Trajectory file
+>    - {% icon param-file %} *"pdb input"*: Structure file
+>    - *"Select domains"*: `Calpha` (calculate RMSF only for the C-alpha domain of the protein)
+>
+{: .hands_on}
+
+![Snapshot of RMSF plot]({{ site.baseurl }}{% link topics/computational-chemistry/images/rmsf_plot_23.png %} "RMSF plot for a short CBH1 simulation")
+
+> > ### {% icon question %} Question
```

Comment:
There are some extra `>` before each line in this box. Could you remove them? Thanks